app/discover/procurement/page.tsx

```diff
@@ -0,0 +1,398 @@
+import Layout from '@/components/Layout'
+import Text from '@/components/Text'
+import { Metadata } from 'next'
+import HighlightBox from '@/components/HighlightBox'
+import Button from '@/components/Button'
+import Link from 'next/link'
+export const metadata: Metadata = {
+  title: 'Instructions for Procurement',
+}
+
+export default function ProcurementPage() {
+  return (
+    <Layout heroSmall heroTitle='Instructions for Procurement'>
+      <div>
+
+<h1>Introduction</h1>
+
+<Text>
+This document was created to answer the need of JDF (Joint Development Forum) to get some help for procuring Oskari. In this document procuring refers to purchasing work related to Oskari. But not only that, the purchasement should be done so that the right supplier is chosen and the contract takes into account the needs of the procurer and Oskari&apos;s general guidelines. And of course the results of the work should enable the procurer organisation to fulfil its objectives related to Oskari.
+</Text>
+
+<Text>
+The instructions have been divided into several chapters. The introduction goes briefly through the background of Oskari and some general thoughts about procuring Oskari. The next three sections tackle three different needs for procuring: procuring a new instance, procuring an update to an existing Oskari instance and procuring customisation to an existing Oskari instance.The aim is to make these processes easier and help the procurer to provide both themselves and the suppliers the information needed for a successful procurement.
+</Text>
+
+<Text>
+The ideas in this document are not set in stone - on the contrary, they are open for discussion and improvement. The author welcomes all constructive criticism and further ideas to develop what is offered here. Hopefully this document gives a general idea of what to take into consideration when procuring something Oskari-related.
```

Comment:
"The author..." here sounds a bit weird as this isn't a blog post or anything.  Maybe just something like "This is a living document and any constructive criticism and further ideas to develop what is offered here is welcomed either as input through the existing channels (link to about/contact us) or even as pull requests"?

Reply:
Good point. Done